release-notes/opensearch-cross-cluster-replication.release-notes-1.3.5.0.md

```diff
@@ -2,6 +2,15 @@
 
 Compatible with OpenSearch 1.3.5
 
+### Bug Fixes
+
+* Fix for missing ShardReplicationTasks on new nodes ([#497](https://github.com/opensearch-project/cross-cluster-replication/pull/497))
+* Updating filters as well during Alias update ([#491](https://github.com/opensearch-project/cross-cluster-replication/pull/491))
+* Fixing the follower checkpoint updation in follower stats API ([#441](https://github.com/opensearch-project/cross-cluster-replication/pull/441))
```

Comment:
Nit: Commit messages should be [imperative](https://medium.com/@steveamaza/how-to-write-a-proper-git-commit-message-e028865e5791).